fluss-common/src/main/java/com/alibaba/fluss/record/MemoryLogRecordsArrowBuilder.java

```diff
@@ -246,12 +251,6 @@ public void deallocate() {
     }
 
     public int getSizeInBytes() {
```

Comment:
This will make the `getSizeInBytes()` inconsistent if `getSizeInBytes()` is called before `close()`. A clean solution:

Add a member variable `boolean reCalculateSizeInBytes = false`, and set it to true for `append`. And re-calculate the `sizeInBytes` in `getSizeInBytes()` if `reCalculateSizeInBytes` is `true`, and set it back to false in the end of `getSizeInBytes()`.